### ISSUE TITLE
Support battery level for Italian Fritz!OS

### ISSUE THREAD

**Issue by giannimp** [andig/homebridge-fritz#99]
Hi, thank you for your work, I Have a problem battery level in homekit 

In homekit the battery level of the heads is not displayed, as an account in the config I enter the administrative one with all the permissions
[Homebridge] [12/30/2019, 4:27:37 PM] [Fritzbox7590] Updating thermostat 099950167xxx
[Homebridge] [12/30/2019, 4:27:37 PM] [Fritzbox7590] Getting thermostat 099950167xxx temperature
[Homebridge] [12/30/2019, 4:27:37 PM] [Fritzbox7590] Getting thermostat 099950167xxx target temperature
[Homebridge] [12/30/2019, 4:27:37 PM] [Fritzbox7590] Getting thermostat 099950167xxx battery level
[Homebridge] [12/30/2019, 4:27:38 PM] [Fritzbox7590] Getting thermostat 099950167xx battery status
[Homebridge] [12/30/2019, 4:27:38 PM] [Fritzbox7590] Updating thermostat 119610841xxx
[Homebridge] [12/30/2019, 4:27:38 PM] [Fritzbox7590] Getting thermostat 119610841xxx temperature
[Homebridge] [12/30/2019, 4:27:38 PM] [Fritzbox7590] Getting thermostat 119610841xxx target temperature
[Homebridge] [12/30/2019, 4:27:38 PM] [Fritzbox7590] Getting thermostat 119610841xxx battery level
[Homebridge] [12/30/2019, 4:27:38 PM] [Fritzbox7590] Getting thermostat 119610841xxx battery status

**Comment by andig** [comment]
Which language of FritzOS are you using?

**Comment by giannimp** [comment]
Italian

**Comment by andig** [comment]
Then thats the problem as its not found by the regex. Could you share a screenshot?

**Comment by giannimp** [comment]
![3C6D7F13-9539-4025-B549-E48AFAA7B136](https://user-images.githubusercontent.com/45820952/71594985-370e4100-2b3a-11ea-9fdb-0a8b84e3dc15.png)
![Uploading A90F94EB-52C3-47AA-9D97-26E98F5606D1.png…]()

**Comment by giannimp** [comment]
![image](https://user-images.githubusercontent.com/45820952/71595020-5efda480-2b3a-11ea-9737-bb3cc730adc9.png)

**Comment by giannimp** [comment]
Hi, Happy new year 2020! Have you news for my issue?

**Comment by bauer-andreas** [comment]
This is probably more an issue of [andig/fritzapi](https://github.com/andig/fritzapi).

Could you simplify your regex pattern to match the percent symbol so any language works @andig?

**Comment by andig** [comment]
Right now we're looking for a label that contains battery: https://github.com/andig/fritzapi/blob/master/index.js#L636

**Comment by giannimp** [comment]
Hi, @andig have you news?

**Comment by andig** [comment]
No, should I? I've shown the line where it happens, please feel try to adapt as needed. If you can find something that will work for Italian *and* other devices I'd be happy to update accordingly.

**Comment by bauer-andreas** [comment]
Could somebody provide a sample html file from the site being parsed @andig?

**Comment by bauer-andreas** [comment]
Any updates? If somebody could send me the html source code of the fritz page the battery percentage is queried from I could maybe come up with a more general regex which supports more languages.

cc @andig @giannimp
